RNWCPP/Playground/Playground/MainPage.xaml.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using react.uwp;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace Playground
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+    const string JSFILENAME = "Playground/Playground/index.uwp";
+    const string JSCOMPONENTNAME = "Playground";
+
+    public MainPage()
+        {
+            this.InitializeComponent();
+            LoadReact();
+    }
+
+    private void LoadReact()
+    {
+      InstanceSettings settings;
+      settings.UseWebDebugger = true;
+      settings.UseLiveReload = true;
+
+      var instance = Instance.Create(JSFILENAME);
+
+      //instantiate sample module for registering callbacks for live reload, JS error handling etc.,
+      instance.RegisterModule(new SampleModule());
+      instance.Start(settings);
+
+      RootElement.Instance = instance;
+
+      //Setup sample initial properties for initializing UI thread
+      string initialProps = "{ "
+          + "\"one\":\"1\""
+          + ", \"two\":\"2\""
+          + "}";
+
+      RootElement.InitialProps = initialProps;
+
+      RootElement.JsComponentName = JSCOMPONENTNAME;
+
+      RootElement.StartRender();
+    }
+  }
+
+  public class SampleModule : react.uwp.IModule
```

Comment:
>SampleModule [](start = 15, length = 12)

Will this code ever do something useful?  If not, remove.  We should probably have separate samples for native modules/custom views.